src/main/java/io/github/thebusybiscuit/exoticgarden/listeners/PlantsListener.java

```diff
@@ -377,6 +379,17 @@ public void onEntityExplode(EntityExplodeEvent e) {
         e.blockList().removeAll(getAffectedBlocks(e.blockList()));
     }
 
+    @EventHandler
```

Comment:
We can ignore already cancelled events anyway.
```suggestion
    @EventHandler(ignoreCancelled = true)
```